scripts/biomedical/chembl/README.md

```diff
@@ -0,0 +1,84 @@
+# Importing ChEMBL proteome data
```

Comment:
Please ensure that the README.md is to standard with other README.md files such as https://github.com/datacommonsorg/data/tree/dbSNP_update/scripts/biomedical/dbSNP

A lot of information is missing and needs to be added to this README.md.

Reply:
A lot of info is still missing from the README.md. This needs to be fixed.